modules/metering-exposing-the-reporting-api.adoc

```diff
@@ -0,0 +1,152 @@
+// Module included in the following assemblies:
+//
+// * metering/configuring-metering/metering-configure-reporting-operator.adoc
+
+[id="metering-exposing-the-reporting-api_{context}"]
+= Exposing the reporting API
+
+On {product-title} the default metering installation automatically exposes a Route, making the reporting API available. This provides the following features:
+
+* Automatic DNS
+* Automatic TLS based on the cluster CA
+
+Also, the default installation makes it possible to use the OpenShift service for serving certificates to protect the reporting API with TLS. The OpenShift OAuth proxy is deployed as a side-car container for `reporting-operator`, which protects the reporting API with authentication.
+
+[id="metering-openshift-authentication_{context}"]
+== Using OpenShift Authentication
+By default, the reporting API is secured with TLS and authentication. This is done by configuring the `reporting-operator` to deploy a pod containing both the `reporting-operator's` container, and a sidecar container running OpenShift auth-proxy.
+
+In order to access the reporting API, the metering operator exposes a route. Once that route has been installed, you can run the command below to get the route's hostname.
+
+[source,bash]
+----
+METERING_ROUTE_HOSTNAME=$(oc -n openshift-metering get routes metering -o json | jq -r '.status.ingress[].host')
+----
+
+Next, set up authentication using either a service account token or basic authentication with a username/password.
+
+=== Authenticate using a service account token
+With this method, you use the token in the reporting Operator's service account, and pass that bearer token to the Authorization header in the following command:
```

Comment:
I need to review our guidelines but it would make more sense to me if it were "Reporting operator" because "Reporting" is the named object of the operator object type.